ontology/yaml/resources/LIGHTING/entity_types/ABSTRACT.yaml

```diff
@@ -136,4 +136,20 @@ LC:
   is_abstract: true
   uses:
   - illuminance_setpoint
-  - illuminance_sensor
\ No newline at end of file
+  - illuminance_sensor
+ 
+EGW:
+  id: "14486067738483352777"
+  description: "Lighting Control Gateway"
+  is_abstract: true
+  uses:
```

Comment:
I'm not a big fan of these all combined into a single functional group, unless you can explain how dimmer fade and zone lighting switch command belong together.

Please explain, or lets create instead some abstract types to cover these.

Reply:
i now see what you mean sir. I will breakdown into features, so that it would be reusable and actually i found some that's already defined in other yaml files within LIGHTING (so no need for me to define those). Sorry for the inconvenience here hehe I will remove this. Thanks